Random Image/app.js

```diff
@@ -0,0 +1,9 @@
+const createImg = document.querySelector('button#randomize');
+
+const randomImg = function(){
+const createNewImg = document.createElement('img');
+createNewImg.setAttribute('src', 'https://source.unsplash.com/random');
```

Comment:
to get a picture with a certain height and width, create new variables that get the values from the DOM
then, use the variables inside the URL

similar to this.

```
let width = document.querySelector("input.width");
let height = document.querySelector("input.height");
let url = ("https://source.unsplash.com/" + width + "x" + height);
let image = document.querySelector(".img");
image.setAttribute("src", url);
```